packages/radix-ui-themes/src/components/radio-card-group.css

```diff
@@ -0,0 +1,132 @@
+.rt-RadioCardGroupRoot {
+  user-select: none;
+  cursor: default;
+}
+
+.rt-RadioCardGroupItem {
+  display: flex;
+  align-items: center;
+  cursor: var(--cursor-radio);
+}
+
+/* Avoid unintentional drag interactions (e.g. on images) */
+.rt-RadioCardGroupItem * {
+  pointer-events: none;
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * SIZES                                                                                           *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+.rt-RadioCardGroupItem {
```

Comment:
Changes from the WorkDS version:
- I removed the fixed height and rely on padding instead
- padding and radius have been aligned with the card ones for the same 3 sizes